### PR TITLE
Tutorial: Setting up SCION on Android devices

### PR DESCRIPTION
**Pull request by stschwar**
This change is [Reviewable](https://reviewable.io/reviews/netsec-ethz/scion-tutorials/19)

### CODE REVIEW

File: content/native_setup/android.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+Install the required packages from within Termux:
+
+```shell
+pkg install -y termux-setup-storage termux-exec ssh git python python2 clang make python-dev libffi-dev openssl-dev openssl-tools
+```
+
+### Configure Go workspace
+
+SCION requires a specific Go version. Termux' Go package is usually ahead of the required version. You can build the required packages yourself (e.g. using [termux-packages](https://github.com/termux/termux-packages)) or use the prebuilt package from the SCION repository:
```
Comment (perrig) [inline content/native_setup/android.md]: Suggestion for minor improvement is to add the text in brackets:
[The] Termux' Go package ...
e.g.[,]

File: content/native_setup/android.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+patch debug.c++ debug.c++.patch
+rm debug.c++.patch
+```
+
+Back in the `capnproto-c++-0.6.1/` root directory run:
+```shell
+./configure TMPDIR=$PREFIX/usr/tmp
+make PREFIX=$PREFIX
+```
+To sucessfully install the libraries, modify the `Makefile` script with an editor (e.g. `vim` or `nano`).
```
Comment (perrig) [inline content/native_setup/android.md]: e.g.[,]

**Review by perrig — APPROVED** [review]
Looks excellent! (I did not run the commands, just read through the tutorial.)

**Comment by hausheer** [comment]
I finally got it working on my Android following the tutorial, but I needed to add and fix a few points, please see https://github.com/netsec-ethz/scion-tutorials/commit/f4944459162f9a7f658ef553897ea432ea695b96

Some further remarks:

- The `zlog-makefile.patch` patch didn't work for me (I applied the changes manually), can you check it?
- `sudo` and `ldconfig` need to be removed from from `sub/lwip-contrib/ports/unix/proj/scion/Makefile`, could you provide a patch for that too?
- Could you add a few lines how to access Termux over ADB?
- To remain in sync with the SCION code, it would be useful to provide a list of changes that need to be applied, rather than a separate SCION repository.

**Comment by hausheer** [comment]
Update: I just saw that you already have the lwip patch in your SCION repo, so I added that to the tutorial in this commit: https://github.com/netsec-ethz/scion-tutorials/commit/dff169df09ad924ee21be59bf214d38699a17484

**Comment by hausheer** [comment]
One more thing: I also fixed `supervisor/supervisor.sh` in the SCION repo as follows:
instead of `if [ ! -e /tmp/supervisor.sock ]; then`
it should be `if [[ $(netstat -lnt | grep 9011) = *listen* ]]; then`
could you fix that in your SCION repo, too?

**Pull request merged**